src/flows/pipes/RawFluxEditor/view.tsx

```diff
@@ -82,10 +82,23 @@ const Query: FC<PipeProp> = ({Context}) => {
         text = `  |> ${fn.example}`
       }
 
+      const getHeader = fn => {
+        let importStatement
+
+        if (fn.package) {
+          importStatement = `import "${fn.package}"`
+          if (isFlagEnabled('fluxDynamicDocs') && fn.path.includes('/')) {
+            importStatement = `import "${fn.path}"`
+          }
+          return importStatement
+        }
+        return null
+      }
+
```

Comment:
Would this version be clearer and easier to read?

```
      const getHeader = fn => {
        let importStatement = null

        if (fn.package) {
          importStatement = `import "${fn.package}"`
          if (isFlagEnabled('fluxDynamicDocs') && fn.path.includes('/')) {
            importStatement = `import "${fn.path}"`
          }
        }
        return importStatement
      }
```

Reply:
oh for sure!

Reply:
oh for sure!